services/network-storage.rst

```diff
@@ -0,0 +1,103 @@
+Network Storage
+===============
+
+The network storage service provides a file store and enables a new kind of
+``mount`` that can be shared between different application containers.
+application containers.
+
+To use it in your application, add it to ``.symfony/services.yaml``:
+
+.. code-block:: yaml
+
+    files:
+        type: network-storage:1.0
+        disk: 256
+
+And wire it in ``.symfony.cloud.yaml``:
+
+.. code-block:: yaml
+
+    mounts:
+        "/var": { source: local, source_path: var }
```

Comment:
```suggestion
        '/var': { source: local, source_path: var }
```

Reply:
@fabpot you may want to update this in the fast track as well